crates/bevy_ecs/src/query/state.rs

```diff
@@ -308,6 +476,24 @@ impl<D: QueryData, F: QueryFilter> QueryState<D, F> {
         }
     }
 
+    fn update_archetypes_with(
```

Comment:
even for private functions, a little documentation would be nice

Reply:
Fair enough, added some